### PR TITLE
Support maximum ADC acquisition time for GD32 ADC

### PR DESCRIPTION
**Pull request by jrwhite**
Handle ADC_ACQ_TIME_MAX acquisition time

### CODE REVIEW

File: drivers/adc/adc_gd32.c [hunk 1]
```diff
@@ -174,7 +174,9 @@ static int adc_gd32_configure_sampt(const struct adc_gd32_config *cfg,
 {
 	uint8_t index = 0, offset;
 
-	if (acq_time != ADC_ACQ_TIME_DEFAULT) {
+        if (acq_time == ADC_ACQ_TIMER_MAX) {
```
Comment (cameled) [inline drivers/adc/adc_gd32.c:177]: You mean `ADC_ACQ_TIME_DEFAULT`? Looks that really helpful if someone just want the biggest one.

Please help to adjust you `tab` setting. Zephyr use the similar code style like Linux,  more detail: https://docs.zephyrproject.org/latest/contribute/guidelines.html#coding-style

**Review by cameled — CHANGES_REQUESTED** [review]
@jrwhite Thanks for the contribution! 

Zephyr have commit guidelines to unify the format of commit message. more detail: https://docs.zephyrproject.org/latest/contribute/guidelines.html#commit-guidelines

**Comment by nandojve** [comment]
Hi @jrwhite ,

I was wondering if you have time to check @cameled comments. It'll be nice if we can have this fix for the next release.
Please, let us know if you still have interest and time to finish the changes.

**Comment by nandojve** [comment]
Hi @jrwhite,

Is there any news here?

**Pull request closed without merging**